actions/issue-reopener/__tests__/verifier.test.ts

```diff
@@ -57,6 +59,143 @@ describe("validateFileDigest", () => {
         tmpFilePath,
         "5aa03f96c77536579166fba147929626cc3a97960e994057a9d80271a736d10f",
       ),
-    ).rejects.toThrowError();
+    ).rejects.toThrow(verifier.FileError);
+  });
+});
+
+describe("downloadSLSAVerifier", () => {
+  const env = process.env;
+
+  beforeEach(() => {
+    jest.resetModules();
+    process.env = { ...env };
+    process.env.RUNNER_TEMP = os.tmpdir();
+  });
+
+  afterEach(() => {
+    process.env = env;
+  });
+
+  it("downloads the verifier", async () => {
+    const verifierPath = await verifier.downloadSLSAVerifier(
```

Comment:
Instead of actually downloading `slsa-verifier` etc. mock the actions toolkit functions and check the arguments.